meson.build

```diff
@@ -23,8 +23,7 @@ libavutil = dependency('libavutil', version: '>= 56.70.100')
 libswresample = dependency('libswresample', version: '>= 3.9.100')
 libswscale = dependency('libswscale', version: '>= 5.9.100')
 
```

Comment:
nitpick, but this empty line is not necessary

Reply:
Well it was to logically separate it from the ffmpeg dependencies I presume.

Reply:
There was originally a space between ffmpeg deps and libass, so I kept that. I think it makes sense

Reply:
I'm stupid, don't mind me. I though they are grouped together.